playbooks/netuitive-agent.yml

```diff
@@ -4,26 +4,30 @@
 
```

Comment:
Thanks @lousyd, this is great! My only suggestion here is that the inline comments reflect the change.  Perhaps instead of 
# - it is necessary to update this playbook with your Linux apikey
we do
# - You will be prompted at run for you Linux apikey

Reply:
Done!